src/shared-components/banner/style.js

```diff
@@ -0,0 +1,92 @@
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import {
+  COLORS,
+  MEDIA_QUERIES,
+  BOX_SHADOWS,
+  SPACING,
+  TYPOGRAPHY_CONSTANTS,
+} from '../../constants';
+
+const successBannerStyles = css`
+  background-color: ${COLORS.statusGreenBackground};
+  border-color: ${COLORS.statusGreenBorder};
+  color: ${COLORS.statusGreen};
+  fill: ${COLORS.statusGreen};
+`;
+
+const errorBannerStyles = css`
+  background-color: ${COLORS.statusRedBackground};
+  border-color: ${COLORS.statusRedBorder};
+  color: ${COLORS.statusRed};
+  fill: ${COLORS.statusRed};
+`;
+
+const defaultBannerStyles = css`
+  background-color: ${COLORS.statusGreyBackground};
+  border-color: ${COLORS.statusGreyBorder};
+  color: ${COLORS.statusGrey};
+  fill: ${COLORS.statusGrey};
+`;
+
+export const BannerContainer = styled.div`
+  align-items: flex-start;
+  border-width: 1px;
+  border-style: solid;
+  box-shadow: ${BOX_SHADOWS.message};
+  cursor: pointer;
+  display: flex;
+  font-size: ${TYPOGRAPHY_CONSTANTS.fontSize.caption};
+  justify-content: space-around;
+  margin: 0 auto ${SPACING.xsmall};
+  padding: ${SPACING.xsmall} 0px;
+  position: relative;
+  text-align: left;
+
+  ${(props => {
+    switch (props.bannerType) {
+      case 'danger':
+        return errorBannerStyles;
+      case 'success':
+        return successBannerStyles;
+      default:
+        return defaultBannerStyles;
+    }
+  })}
+
+  ${MEDIA_QUERIES.lgUp} {
+    font-size: ${TYPOGRAPHY_CONSTANTS.fontSize.caption};
+    margin-bottom: ${SPACING.small};
+    padding: ${SPACING.xsmall} 0px;
+  }
+`;
+
+export const BannersContainer = styled.div`
```

Comment:
Not entirely sold that we need this. Banners currently are being used in the flow of the natural dom, whereas the alert needs this container.